app/services/decisions/client_decision_tree.rb

```diff
@@ -133,18 +131,6 @@ def determine_dwp_result_page
       end
     end
 
-    def after_dwp_check
-      edit('/steps/case/urn')
-    end
-
-    def after_has_benefit_evidence
-      if form_object.has_benefit_evidence.yes?
-        edit('/steps/case/urn')
-      else
-        show(:evidence_exit)
```

Comment:
We're not going to forget to update the copy on the 'No' option on this page right? There is that other ticket for it

Reply:
Yes there is but i just added it quickly

<img width="1034" alt="Screenshot 2024-04-04 at 12 26 20" src="https://github.com/ministryofjustice/laa-apply-for-criminal-legal-aid/assets/51047911/75179768-a64a-4afb-bdef-99f68a76b060">